src/Resolvers/Collections.ts

```diff
@@ -92,31 +92,42 @@ export class CollectionsResolver {
 
   @FieldResolver(type => [Collection])
   async relatedCollections(
+    @Arg("size", () => Int, { nullable: true }) size: number,
     @Root() collection: Collection
   ): Promise<Collection[]> {
+    const query: { where: {}; take?: number } = { where: {} }
     const isArtistBased =
       collection.query.artist_ids && collection.query.artist_ids.length > 0
 
+    if (size) {
+      query.take = size + 1
```

Comment:
I _think_ this is taking `size + 1` because we might end up filtering out the original collection from the results. Whether I'm right or wrong, I think we should add a comment here so readers aren't confused about why we're taking one extra.

Reply:
This does feel a bit roundabout! Maybe instead we can limit to the size we want but explicitly exclude the current collection's ID from the results (via our query)?